tools/src/icon4pytools/py2fgen/template.py

```diff
@@ -243,7 +243,6 @@ class PythonWrapperGenerator(TemplatedGenerator):
 {% if _this_node.backend == 'GPU' %}import cupy as cp {% endif %}
 from numpy.typing import NDArray
 from gt4py.next.iterator.embedded import np_as_located_field
```

Comment:
Is this missing here? `import gt4py.next as gtx`

Reply:
`np_as_located_field` will be deprecated soon so I would leave as is now, plus this is within a docstring and it's trickier to make edits such that tests will pass